src/Cake.Jira/Cake.Jira.csproj

```
@@ -24,13 +24,10 @@
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Atlassian.SDK" Version="10.4.0" />
-    <PackageReference Include="Cake.Core" Version="0.33.0" PrivateAssets="All" />
+    <PackageReference Include="Atlassian.SDK" Version="12.4.0" />
+    <PackageReference Include="Cake.Core" Version="1.1.0" PrivateAssets="All" />
```

```suggestion
    <PackageReference Include="Cake.Core" Version="1.0.0" PrivateAssets="All" />
```

We recommend referencing the lowest possible version of Cake.Core for broader compatibility: https://cakebuild.net/docs/extending/addins/best-practices#cake-reference

By referencing Cake.Core 1.1.0 you'll be preventing users still on Cake 1.0.0 from using this addin.